lib/builder/builder.js

```diff
@@ -242,6 +242,7 @@ export default class Builder extends Tapable {
           hash,
           r,
           wp,
+          resolveModule,
```

Comment:
For resolving modules there is already a helper function [resolvePath](https://github.com/nuxt/nuxt.js/blob/dev/lib/core/nuxt.js#L101) in code doing same. So we may change this to:
```js
resolvePath: this.nuxt. resolvePath.bind(this.nuxt)
```